### DISCUSSION TITLE
Cyrillic characters support for osu! fonts

### DISCUSSION THREAD

**Discussion by CaXaPeK** [ppy/osu#23229] [Ideas]
There's been a problem with the game's fonts that they don't support Cyrillic characters. So, in many languages, the text often looks not quite as good and appealing as in Latin languages.

![image](https://user-images.githubusercontent.com/70052397/232757435-5557c821-680e-4b98-9b51-36b38a2f3572.png)

I could ask a friend of mine to create stylized Cyrillic characters for both types of the Torus font and (probably) Venera. She is quite experienced at this kind of work — she had created/adjusted Cyrillic characters for many Russian fan translations in the past — so I'm sure she'll do a good job this time as well. With good fonts the UX would be much more pleasant!

So, would you need this kind of contribution?

**Comment by nagi-desuuu** [comment]
Don't you also need to contact the original font creators, too?

**Reply by CaXaPeK** [reply]
Sure, but first I need the developers or someone to tell me if making a look-alike font is necessary right now. What if the font is going to be changed in some future update?
